Exercise_A/assets/css/style.css

```diff
@@ -1,3 +1,491 @@
-/* 
-Add in your custom style such that you can develop the wonderful website within the exercise. 
-*/
+/*!
+ * Start Bootstrap - Creative Bootstrap Theme (http://startbootstrap.com)
+ * Code licensed under the Apache License v2.0.
+ * For details, see http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+ html,
+ body {
+     width: 100%;
+     height: 100%;
+     position: relative;
+  
+ }
+ 
+ body {
+     font-family: Merriweather,'Helvetica Neue',Arial,sans-serif;
+ }
+ a{
+     text-decoration: none!important;
+ }
+ 
+ hr {
+     max-width: 50px;
+     border-color: #f05f40;
+     border-width: 3px;
+ }
+ 

```

Comment:
Brilliant use of this style.

Fix the positioning of these hr with margin: auto